assets/queries/cloudFormation/amazonMQ_broker_without_encryption_options/metadata.json

```diff
@@ -0,0 +1,8 @@
+{
+  "id": "amazonMQ_broker_without_encryption_options",
+  "queryName": "AmazonMQ Broker Without EncryptionOptions",
```

Comment:
change query id to `amazonmq_broker_encryption_disabled`

also change the query directory name to: `amazonmq_broker_encryption_disabled`

queryName = `AmazonMQ broker encryption disabled`